Doc/library/datetime.rst

```diff
@@ -295,6 +295,20 @@ Instance attributes (read-only):
 
    Between 0 and 86,399 inclusive.
 
+   .. warning::
+
+      It is a somewhat common bug for code to unintentionally use this attribute
+      when it actually intended to get a :meth:`~timedelta.total_seconds` value
```

Comment:
```suggestion
      when it is actually intended to get a :meth:`~timedelta.total_seconds` value
```

Reply:
link to change section for anyone else looking: https://cpython-previews--124811.org.readthedocs.build/en/124811/library/datetime.html#datetime.timedelta.seconds